projects/js-packages/api/index.jsx

```diff
@@ -92,9 +92,10 @@ function JetpackRestApiClient( root, nonce ) {
 
 		fetchAuthorizationUrl: redirectUri =>
 			getRequest(
-				`${ apiRoot }jetpack/v4/connection/authorize_url?no_iframe=1&redirect_uri=${ encodeURIComponent(
-					redirectUri
-				) }`,
+				addQueryString(
```

Comment:
Would it be an option to rely on `addQueryArgs` from `@wordpress/url` here?

Reply:
Good point, I'll rewrite that.

Reply:
Done 👍